build.gradle

```diff
@@ -10,10 +10,11 @@ apply plugin: 'java-gradle-plugin'
 apply plugin: 'maven-publish'
 apply from: "$rootDir/gradle/functional-test.gradle"
 
+group = 'com.tomtom.gradle'
 
 gradlePlugin {
   plugins {
-    updateChangelogPlugin {
+    updateChangelog {
```

Comment:
I can see in all the examples that the `Plugin` suffix is used, is it automagically added somehow now?
Just wondering if using [Java Gradle Development Plugin](https://docs.gradle.org/current/userguide/custom_plugins.html#sec:using_the_java_gradle_plugin_development_plugin) could eliminate some boilerplate in this case?

Reply:
The `Java Gradle Development Plugin` is already used, see: https://github.com/tomtom-international/updatechangelog-gradle-plugin/blob/master/build.gradle#L9
Actually, tasks and publication names which this plugin generates made me remove the `Plugin` suffix here. 
When plugin is applied it generates two publications now: `pluginMaven` and `updateChangelogPluginMarkerMaven`. With `Plugin` suffix, second maven publication was called `updateChangelogPluginPluginMarkerMaven`. 
Since this project uses `bintray` publication instead of public Gradle plugins repository, I needed to mention the long publication name as well. And the name looks better without duplicated words.
Examples do use the `Plugin` suffix as most projects use `publish` task generated by Maven publishing plugin. This task depends on both plugin and marker publication, while publication tasks generated by `bintray` plugin require explicit publications names.